acvm-repo/brillig_vm/src/memory.rs

```diff
@@ -127,9 +184,12 @@ impl<F: std::fmt::Display> std::fmt::Display for MemoryValue<F> {
     fn fmt(&self, f: &mut ::std::fmt::Formatter) -> Result<(), ::std::fmt::Error> {
         match self {
             MemoryValue::Field(value) => write!(f, "{}: field", value),
-            MemoryValue::Integer(value, bit_size) => {
-                write!(f, "{}: {}", value, bit_size)
-            }
+            MemoryValue::U1(value) => write!(f, "{}: {}", value, 1),
+            MemoryValue::U8(value) => write!(f, "{}: {}", value, 8),
+            MemoryValue::U16(value) => write!(f, "{}: {}", value, 16),
+            MemoryValue::U32(value) => write!(f, "{}: {}", value, 32),
+            MemoryValue::U64(value) => write!(f, "{}: {}", value, 64),
+            MemoryValue::U128(value) => write!(f, "{}: {}", value, 128),
```

Comment:
```suggestion
            MemoryValue::U1(value) => write!(f, "{}: bool", value),
            MemoryValue::U8(value) => write!(f, "{}: u{}", value, 8),
            MemoryValue::U16(value) => write!(f, "{}: u{}", value, 16),
            MemoryValue::U32(value) => write!(f, "{}: u{}", value, 32),
            MemoryValue::U64(value) => write!(f, "{}: u{}", value, 64),
            MemoryValue::U128(value) => write!(f, "{}: u{}", value, 128),
```

Alternatively you could recover the `IntegerBitSize` which supports `Display` using `MemoryValue::bit_size`.

Reply:
I've inlined the constants entirely here. I've also gone for `u1` over `bool` as the brillig VM doesn't really have a concept for bools.